arrow/arrow.py

```diff
@@ -692,6 +692,25 @@ def timestamp(self):
 
         """
 
+        warnings.warn(
+            "For compatibility with the datetime.timestamp() method this property will be replaced with a method in "
+            "the 1.0.0 release, please switch to .int_timestamp for identical behaviour as soon as possible.",
```

Comment:
Should we say "please switch to the int_timestamp property as soon as possible"? This way it is clear that we are replacing a property with another property?